src/main/java/org/folio/util/ResponseUtils.java

```diff
@@ -24,21 +24,19 @@ public class ResponseUtils {
   private ResponseUtils() {
   }
 
-  public static <T> Handler<AsyncResult<Tx<T>>> handleNoContentResponse(Handler<AsyncResult<Response>> asyncResultHandler, Tx<T> tx,
-    String logMessage) {
-    logger.debug("handleNoContentResponse:: Trying to handle no content response for entity: {}', logMessage: {}", tx.getEntity(), logMessage);
-    return result -> {
-      if (result.failed()) {
-        HttpException cause = (HttpException) result.cause();
-        logger.error(logMessage, cause, tx.getEntity(), "or associated data failed to be");
-
-        // The result of rollback operation is not so important, main failure cause is used to build the response
-        tx.rollbackTransaction().onComplete(res -> asyncResultHandler.handle(buildErrorResponse(cause)));
-      } else {
-        logger.info(logMessage, tx.getEntity(), "and associated data were successfully");
-        asyncResultHandler.handle(buildNoContentResponse());
-      }
-    };
+  public static void handleNoContentResponse(AsyncResult<Conn> result,
+                                             String id,
+                                             String logMessage,
+                                             Handler<AsyncResult<Response>> onComplete) {
+    logger.debug("handleNoContentResponse:: Trying to handle no content response for entity: {}', logMessage: {}", id, logMessage);
+    if (result.failed()) {
+      HttpException cause = (HttpException) result.cause();
+      logger.error(logMessage, cause, id, "or associated data failed to be");
```

Comment:
this log looks over complicated and used only in our place, I think we can simplify

Reply:
Removed this log since there are more containing the same information